sound/soc/sof/amd/Kconfig

```diff
@@ -17,14 +17,23 @@ if SND_SOC_SOF_AMD_TOPLEVEL
 config SND_SOC_SOF_AMD_COMMON
 	tristate
 	select SND_SOC_SOF
+	select SND_SOC_SOF_PCI_DEV
+	select SND_AMD_ACP_CONFIG
+	select SND_SOC_ACPI if ACPI
 	help
 	  This option is not user-selectable but automatically handled by
 	  'select' statements at a higher level
 
+config SND_SOC_SOF_PCI_RENOIR
+	tristate
+	select SND_AMD_ACP_CONFIG
+	select SND_SOC_SOF_AMD_COMMON
+	help
+	  This option get enabled with SOF Renoir support.
+
 config SND_SOC_SOF_AMD_RENOIR
 	tristate "SOF support for RENOIR"
-	select SND_SOC_SOF_AMD_COMMON
+	select SND_SOC_SOF_PCI_RENOIR
```

Comment:
unclear to me why you have an indirection? You could remove the PCI_RENOIR and move the selects here